qutip/tests/solver/test_flimesolve.py

```diff
@@ -0,0 +1,312 @@
+import numpy as np
+from qutip import (
+    sigmax,
+    sigmay,
+    sigmaz,
+    rand_ket,
+    num,
+    destroy,
+    mesolve,
+    expect,
+    Qobj,
+    QobjEvo,
+    basis,
+    correlation,
+)
+
+from qutip.solver.floquet import FloquetBasis
+
+from qutip.solver.flimesolve import FLiMESolver, flimesolve
+import pytest
+
+
+# Writing this script to closely mirror test_floquet.py, but without
+#    anything already tested there (e.g. unitary evolution)
+
+
+class TestFlimesolve:
+    """
+    A test class for the QuTiP functions for Floquet formalism.
+    """
+
+    def testFloquetLindbladMasterEquation1(self):
+        """
+        Test Floquet-Lindblad Master Equation for a driven two-level system
+        without dissipation.
+        """
+        delta = 1.0 * 2 * np.pi
+        eps0 = 1.0 * 2 * np.pi
+        A = 0.5 * 2 * np.pi
+        omega = 100 * np.sqrt(delta**2 + eps0**2)
+        T = (2 * np.pi) / omega
+        tlist = np.linspace(0.0, 2 * T, 101)
+        psi0 = rand_ket(2)
+        H0 = -eps0 / 2.0 * sigmaz() - delta / 2.0 * sigmax()
+        H1 = A / 2.0 * sigmax()
+        args = {"w": omega}
+        H = [H0, [H1, lambda t, args: np.sin(args["w"] * t)]]
+        e_ops = [num(2)]
+        gamma1 = 0
+
+        # Collapse operator for Floquet-Markov Master Equation
+        c_op = sigmax()
+
+        # Solve the floquet-Lindblad master equation
+
+        p_ex = flimesolve(
+            H,
+            psi0,
+            tlist,
+            T,
+            c_ops=[c_op * np.sqrt(gamma1)],
+            e_ops=e_ops,
+            args=args,
+            Nt=50,
```

Comment:
You added a none default value for `Nt` in every test.
Are test failing with the default value or it is there just to ensure the parameter is tested.